plugins/modules/device_credential_workflow_manager.py

```diff
@@ -297,6 +299,57 @@
               id:
                 description: snmp_v3 Credential Id. Use Description or Id.
                 type: str
+      apply_credentials_to_site:
```

Comment:
Why do we need the sub-section "apply_credentials_to_site"? I only see that the 'site_name' is different. Can't we give the site_name when you create/update credentials? If the site_name is there, then create the credential and apply it to the site. If it is not there, then ignore it.

Reply:
Removed apply_credentials_to_site. introduced a new variable "Sync " inside assigned_credentials_to_site, which is of boolean type.